src/apps/explorer/components/OrderWidget/OrderDetails.tsx

```diff
@@ -8,36 +8,25 @@ import { RawOrder } from 'api/operator'
 import { SimpleTable } from 'components/common/SimpleTable'
 
 // TODO: move to theme AAAAND pick color for white variant
-const COLUMN_COLOR = '#252535'
+const FONT_COLOR_DESCRIPTION = 'rgba(255, 255, 255, 0.9);';
+const TABLE_BORDER_COLOR = 'rgba(151, 151, 184, 0.3)';
```

Comment:
we should avoid mixing too many floating constants and theme... try to use the theme whereever possible please

Reply:
Moved to using theme vars.